src/objects/featured-row/featured-row.scss

```
@@ -0,0 +1,47 @@
+@use "../../compiled/tokens/scss/breakpoint";
+@use "../../compiled/tokens/scss/size";
+@use '../../mixins/fluid';
+@use '../../mixins/media-query';
+@use "../../mixins/ms";
+
+.o-featured-row {
+  display: grid;
+  @include fluid.grid-gap(
+    breakpoint.$s,
+    breakpoint.$xl,
+    size.$spacing-gap-fluid-min,
+    size.$spacing-gap-fluid-max
+  );
+  grid-template-areas:
+    'intro'
+    'content'
+    'action';
+  grid-template-rows: repeat(3, minmax(0, auto));
+
+  @media (min-width: breakpoint.$l) {
+    grid-template-areas:
+      'intro content content'
+      '. content content'
```

I think I made a mistake, because as-is it looks like the `action` is always _below_ the content. I think this might work better:

```suggestion
      'intro content content'
```